### PR TITLE
RDM-6470 feature file and json files for F-079 get case type by id

### PR DESCRIPTION
**Pull request by RebeccaBaker**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RDM-6470

### Change description ###
Functional test for get case type by id 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: aat/src/aat/resources/features/F-079/F-079.feature [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  Scenario: must return the case type for an appropriate request
+    Given a user with [an active profile in CCD]
+    When a request is prepared with appropriate values
+    And the request [contains a valid case type id]
+    And it is submitted to call the [fetch a Case Type Schema] operation of [CCD Definition Store]
+    Then a positive response is received
+    And the response [has the 200 OK code]
+    And the response has all other details as expected
+
+  @S-509  @Ignore # Response code mismatch, expected: 401, actual: 403
```
Comment (veeresha-hmcts) [inline aat/src/aat/resources/features/F-079/F-079.feature]: best to mention the Jira ticket here

File: aat/src/aat/resources/features/F-079/F-079.feature [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  Scenario: must return 401 when request does not provide valid authentication credentials
+    Given a user with [an active profile in CCD]
+    When a request is prepared with appropriate values
+    And the request [contains an invalid authentication credentials]
+    And it is submitted to call the [fetch a Case Type Schema] operation of [CCD Definition Store]
+    Then a negative response is received
+    And the response [contains 401 unauthorised code]
+    And the response has all the details as expected
+
+  @S-510 @Ignore # Response code mismatch, expected: 403, actual: 200
```
Comment (veeresha-hmcts) [inline aat/src/aat/resources/features/F-079/F-079.feature]: Same, Jira ticket for this issue

File: aat/src/aat/resources/features/F-079/F-079.feature (hunk 1 above)
Comment (RebeccaBaker) [reply aat/src/aat/resources/features/F-079/F-079.feature]: 👍

File: aat/src/aat/resources/features/F-079/F-079.feature (hunk 2 above)
Comment (RebeccaBaker) [reply aat/src/aat/resources/features/F-079/F-079.feature]: 👍

File: aat/src/aat/resources/features/F-079/F-079.feature [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  Background:
+    Given an appropriate test context as detailed in the test data source
+
+  @S-508 #RDM-6858
+  Scenario: must return the case type for an appropriate request
+    Given a user with [an active profile in CCD]
+    When a request is prepared with appropriate values
+    And the request [contains a valid case type id]
+    And it is submitted to call the [fetch a Case Type Schema] operation of [CCD Definition Store]
+    Then a positive response is received
```
Comment (MSancaktutar) [inline aat/src/aat/resources/features/F-079/F-079.feature]: this line is redundant with the one under it. or vice versa.

File: aat/src/aat/resources/features/F-079/F-079.feature [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    And it is submitted to call the [fetch a Case Type Schema] operation of [CCD Definition Store]
+    Then a negative response is received
+    And the response [contains 403 forbidden code]
+    And the response has all the details as expected
+
+  @S-507
+  Scenario: must return 404 when request provides an invalid case type id
+    Given a user with [an active profile in CCD]
+    When a request is prepared with appropriate values
+    And it is submitted to call the [fetch a Case Type Schema] operation of [CCD Definition Store]
```
Comment (MSancaktutar) [inline aat/src/aat/resources/features/F-079/F-079.feature]: add a line about the request to describe what makes this scenario different to return a 404.

File: aat/src/aat/resources/features/F-079/S-508.td.json [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+      "name": "Demo Case BEFTA CASETYPE 2 1",
+      "jurisdiction": {
+        "id": "BEFTA_JURISDICTION_2",
+        "name": "BEFTA JURISDICTION 2",
+        "description": "Content for the BEFTA Jurisdiction 2.",
+        "live_from": "2017-01-01T00:00:00.000+0000",
+        "live_until": null,
+        "case_types": []
+      },
+      "events": "[[ANY]]",
```
Comment (MSancaktutar) [inline aat/src/aat/resources/features/F-079/S-508.td.json]: Please use fw version 1.0.22. And this. [[ANY]] will need to be changed with 1.0.22.

**Pull request merged**